README.md

```diff
@@ -13,7 +13,8 @@ Details can be found in the [Web Editing Working Group Charter](https://www.w3.o
 * [Clipboard APIs](https://www.w3.org/TR/clipboard-apis/)
 * [Input Events](https://w3c.github.io/input-events/)
 * [Content Editable Disabled](https://w3c.github.io/editing/docs/contentEditableDisabled/)
-* [Virtual Keyboard Policy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/VirtualKeyboardPolicy/explainer.md)
+* [Virtual Keyboard API](https://w3c.github.io/editing/docs/virtualkeyboard/index.html)
+* [Virtual Keyboard Policy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/VirtualKeyboardPolicy/explainer.md)
```

Comment:
We have merged both policy / API explainers for virtual keyboard into one spec called Virtual Keyboard API.  Delete the Virtual Keyboard Policy line; I think the Virtual Keyboard API link above covers it.